### PR TITLE
Ui component only

### PR DESCRIPTION
**Pull request by charithadeshapriya**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by udarakr — APPROVED** [review]

File: components/ui-component/pom.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (sachi-d) [inline components/ui-component/pom.xml:3]: Update the year.

File: components/ui-component/public/images/logo-inverse.svg [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (sachi-d) [inline components/ui-component/public/images/logo-inverse.svg:4]: Update the year.

File: components/ui-component/public/images/logo.svg [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (sachi-d) [inline components/ui-component/public/images/logo.svg:4]: Update the year.

File: components/ui-component/public/images/loading.svg [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (sachi-d) [inline components/ui-component/public/images/loading.svg:4]: Update the year.

**Pull request merged**